src/apps/saddle/evmos/saddle.mini-chef-v2-farm.contract-position-fetcher.ts

```diff
@@ -0,0 +1,76 @@
+import { Inject } from '@nestjs/common';
+
+import { APP_TOOLKIT, IAppToolkit } from '~app-toolkit/app-toolkit.interface';
+import { Register } from '~app-toolkit/decorators';
+import { RewardRateUnit } from '~app-toolkit/helpers/master-chef/master-chef.contract-position-helper';
+import { PositionFetcher } from '~position/position-fetcher.interface';
+import { ContractPosition } from '~position/position.interface';
+import { Network } from '~types/network.interface';
+
+import { SaddleContractFactory } from '../contracts';
+import { SaddleMiniChefV2 } from '../contracts/ethers/SaddleMiniChefV2';
+import { SaddleMiniChefV2Rewarder } from '../contracts/ethers/SaddleMiniChefV2Rewarder';
+import { SADDLE_DEFINITION } from '../saddle.definition';
+
+const appId = SADDLE_DEFINITION.id;
+const groupId = SADDLE_DEFINITION.groups.miniChefV2.id;
+const network = Network.EVMOS_MAINNET;
+
+@Register.ContractPositionFetcher({ appId, groupId, network })
+export class EthereumSaddleMiniChefV2FarmContractPositionFetcher implements PositionFetcher<ContractPosition> {
```

Comment:
This needs to be corrected: Should be Evmos.

Reply:
Yes, rename to `EvmosSaddleMiniChefV2FarmContractPositionFetcher`

Reply:
Completed.